pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -129,6 +83,22 @@ func (p *Protocol) SendExchangeResponse(exchangeResponse *Response, destination
 	return err
 }
 
+//CreateInvitation creates invitation
+//TODO to be implemented
+func (p *Protocol) CreateInvitation() (*CreateInvitationResponse, error) {
```

Comment:
Name on return parameter type is unexpected - this method "creates invitations" yet I receive a response back.

Is this method supposed to create and send an invitation?

Actually, I believe this method belongs in the `service` package. The `exchange` package is a DIDComm package, the `service` package is purely about REST API and models.